qiskit_machine_learning/algorithms/classifiers/pegasos_qsvc.py

```diff
@@ -27,11 +27,12 @@
 logger = logging.getLogger(__name__)
 
 
-class PegasosQSVC(SVC, SerializableModelMixin):
-    """
+class PegasosQSVC(ClassifierMixin, SerializableModelMixin):
+    r"""
     This class implements Pegasos Quantum Support Vector Classifier algorithm developed in [1]
-    and includes overridden methods ``fit`` and ``predict`` from the ``SVC`` super-class. This
-    implementation is adapted to work with quantum kernels.
+    and includes methods ``fit``, ``predict`` and ``decision_function`` following the signatures
+    of `sklearn.svm.SVC <https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html>`.
```

Comment:
```suggestion
    of `sklearn.svm.SVC <https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html>`_.
```
This does not format out as expected as a link - missing the trailing underscore